src/Window.cpp

```diff
@@ -689,19 +689,23 @@ void CWindow::applyDynamicRule(const SWindowRule& r) {
             Debug::log(ERR, "Rule idleinhibit: unknown mode {}", IDLERULE);
     } else if (r.szRule.starts_with("maxsize")) {
         try {
-            m_sAdditionalConfigData.maxSize = configStringToVector2D(r.szRule.substr(8));
-            m_vRealSize                     = Vector2D(std::min((double)m_sAdditionalConfigData.maxSize.toUnderlying().x, m_vRealSize.goal().x),
-                                                       std::min((double)m_sAdditionalConfigData.maxSize.toUnderlying().y, m_vRealSize.goal().y));
-            g_pXWaylandManager->setWindowSize(this, m_vRealSize.goal());
-            setHidden(false);
+            if (m_bIsFloating) {
```

Comment:
please prefer guards

if (floating)
continue;

Reply:
I assume you mean return